Fix TypeError [ERR_INVALID_URL_SCHEME] in function-no-unknown

### What steps are needed to reproduce the bug?

I think there's a bug in 14.5.0. If I use
```
@use "sass:color"
@use "~mypackage"
```

Then I get an error
```
TypeError [ERR_INVALID_URL_SCHEME]: The URL must be of scheme file
    at Object.openSync (fs.js:490:10)
    at Object.readFileSync (fs.js:394:35)
    at \node_modules\stylelint\lib\rules\function-no-unknown\index.js:48:39  
```

### What Stylelint configuration is needed to reproduce the bug?

any

### How did you run Stylelint?

stylelint './src/**/*.scss' --fix

### Which version of Stylelint are you using?

14.5.0

### What did you expect to happen?

No script error

### What actually happened?

```
TypeError [ERR_INVALID_URL_SCHEME]: The URL must be of scheme file
    at Object.openSync (fs.js:490:10)
    at Object.readFileSync (fs.js:394:35)
    at \node_modules\stylelint\lib\rules\function-no-unknown\index.js:48:39  
```

### Does the bug relate to non-standard syntax?

_No response_

### Proposal to fix the bug

Compare with 14.4.0, it only occurs in 14.5.0

Error, only on Windows (via GitHub Actions CI builds):

Stylelint `14.5.0`

```
TypeError [ERR_INVALID_URL_SCHEME]: The URL must be of scheme file
    at Object.openSync (node:fs:577:10)
    at Object.readFileSync (node:fs:453:35)
    at D:\a\project\node_modules\stylelint\lib\rules\function-no-unknown\index.js:48:39
    at D:\a\project\node_modules\stylelint\lib\lintPostcssResult.js:113:8
    at Array.map (<anonymous>)
    at lintPostcssResult (D:\a\project\node_modules\stylelint\lib\lintPostcssResult.js:104:18)
    at lintSource (D:\a\project\node_modules\stylelint\lib\lintSource.js:91:8)
    at async D:\a\project\node_modules\stylelint\lib\standalone.js:227:27
    at async Promise.all (index 0)
    at async standalone (D:\a\project\node_modules\stylelint\lib\standalone.js:266:22)
Error: Process completed with exit code 1.
```

Configuration:

```
{
    "extends": "stylelint-config-standard",
    "rules": {
        "declaration-block-no-redundant-longhand-properties": null,
        "shorthand-property-no-redundant-values": null,
        "selector-class-pattern": null,
        "custom-property-pattern": null,
        "selector-id-pattern": null,
        "keyframes-name-pattern": null,
        "property-no-vendor-prefix": null,
        "value-no-vendor-prefix": null,
        "number-max-precision": null,
        "color-function-notation": null,
        "alpha-value-notation": null,
        "string-quotes": null,
        "selector-attribute-quotes": null,
        "function-url-quotes": null,
        "font-family-name-quotes": null,
        "no-descending-specificity": null,
        "at-rule-no-unknown": [
            true,
            {
                "ignoreAtRules": ["define-mixin", "mixin", "include"]
            }
        ],
        "selector-pseudo-class-no-unknown": [
            true,
            {
                "ignorePseudoClasses": ["global"]
            }
        ]
    }
}
```

--- a/lib/rules/function-no-unknown/index.js
+++ b/lib/rules/function-no-unknown/index.js
@@ -1,7 +1,6 @@
 'use strict';
 
 const fs = require('fs');
-const { URL } = require('url');
 const valueParser = require('postcss-value-parser');
 const functionsListPath = require('css-functions-list');
 
@@ -44,8 +43,7 @@ const rule = (primary, secondaryOptions) => {
 			return;
 		}
 
-		const path = new URL(functionsListPath.toString(), 'file://');
-		const functionsList = JSON.parse(fs.readFileSync(path, 'utf8'));
+		const functionsList = JSON.parse(fs.readFileSync(functionsListPath.toString(), 'utf8'));
 
 		root.walkDecls((decl) => {
 			const { value } = decl;

--- a/package.json
+++ b/package.json
@@ -111,7 +111,7 @@
     "balanced-match": "^2.0.0",
     "colord": "^2.9.2",
     "cosmiconfig": "^7.0.1",
-    "css-functions-list": "^3.0.0",
+    "css-functions-list": "^3.0.1",
     "debug": "^4.3.3",
     "execall": "^2.0.0",
     "fast-glob": "^3.2.11",

